docs/source/changelog.rst

```diff
@@ -17,6 +17,7 @@ Changelog
         * Change default branch to ``main`` (:pr:`1038`)
         * Raise TypeError if improper input is supplied to ``Entity.delete_variables()`` (:pr:`1064`)
         * Updates for compatibility with pandas 1.1.0 (:pr:`1079`, :pr:`1089`)
+        * Upgrade pandas to >=1.1.0 (:pr: `1094`)
```

Comment:
Also, remove the space between `:pr:` and ``1094``

Reply:
Ah that's what it was, thank you!